index.d.ts

```diff
@@ -0,0 +1,1100 @@
+// Type definitions for [~THE LIBRARY NAME~] [~OPTIONAL VERSION NUMBER~]
+// Project: [~THE PROJECT NAME~]
+// Definitions by: [~YOUR NAME~] <[~A URL FOR YOU~]>
+
+export as namespace Draggable;
+
+// type definitions
+export type DraggableContainer = HTMLElement[] | NodeList | HTMLElement
+
+type DraggableEventName = 'draggable:initialized' | 'draggable:destroy' | string
```

Comment:
Do these actually allow any string? I think this should just be the specific event name strings.